src/ResourceManager/DeviceProvisioningServices/Commands.DeviceProvisioningServices/Commands.DeviceProvisioningServices.csproj

```diff
@@ -0,0 +1,162 @@
+    <Compile Include="Cmdlet\IotDpsLinkedHubs\RemoveAzureRmIotDeviceProvisioningServiceLinkedHub.cs" />
+    <Compile Include="Cmdlet\IotDpsLinkedHubs\UpdateAzureRmIotDeviceProvisioningServiceLinkedHub.cs" />
+    <Compile Include="Cmdlet\IotDpsSharedAccessPolicies\AddAzureRmIotDeviceProvisioningServiceAccessPolicy.cs" />
+    <Compile Include="Cmdlet\IotDpsSharedAccessPolicies\GetAzureRmIotDeviceProvisioningServiceAccessPolicy.cs" />
+    <Compile Include="Cmdlet\IotDpsSharedAccessPolicies\RemoveAzureRmIotDeviceProvisioningServiceAccessPolicy.cs" />
+    <Compile Include="Cmdlet\IotDpsSharedAccessPolicies\UpdateAzureRmIotDeviceProvisioningServiceAccessPolicy.cs" />
+    <Compile Include="Cmdlet\IotDps\UpdateAzureRmIotDeviceProvisioningService.cs" />
+    <Compile Include="Common\IotDpsBaseCmdlet.cs" />
+    <Compile Include="Common\IotDpsUtils.cs" />
+    <Compile Include="Cmdlet\IotDps\GetAzureRmIotDeviceProvisioningService.cs" />
+    <Compile Include="Cmdlet\IotDps\RemoveAzureRmIotDeviceProvisioningService.cs" />
+    <Compile Include="Models\PSAccessRightsDescription.cs" />
+    <Compile Include="Models\PSAllocationPolicy.cs" />
+    <Compile Include="Models\PSCertificate.cs" />
+    <Compile Include="Models\PSCertificateProperties.cs" />
+    <Compile Include="Models\PSCertificateResponse.cs" />
+    <Compile Include="Models\PSIotDpsPropertiesDescription.cs" />
+    <Compile Include="Models\PSIotDpsSku.cs" />
+    <Compile Include="Models\PSIotDpsSkuInfo.cs" />
+    <Compile Include="Models\PSIotHubDefinitionDescription.cs" />
+    <Compile Include="Models\PSIotHubDefinitions.cs" />
+    <Compile Include="Models\PSPage.cs" />
+    <Compile Include="Models\PSProvisioningServiceDescription.cs" />
+    <Compile Include="Models\PSProvisioningServicesDescription.cs" />
+    <Compile Include="Models\PSResource.cs" />
+    <Compile Include="Models\PSSharedAccessSignatureAuthorizationRuleAccessRights.cs" />
+    <Compile Include="Models\PSSharedAccessSignatureAuthorizationRuleAccessRightsDescription.cs" />
+    <Compile Include="Models\PSTagsResource.cs" />
+    <Compile Include="Models\PSVerificationCodeResponse.cs" />
+    <Compile Include="Models\PSVerificationCodeResponseProperties.cs" />
+    <Compile Include="Cmdlet\IotDps\NewAzureRmIotDeviceProvisioningService.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\Resources.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="..\AzureRM.DeviceProvisioningServices.psd1">
+      <Link>AzureRM.DeviceProvisioningServices.psd1</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="help\New-AzureRmIoTDeviceProvisioningServiceCertificateVerificationCode.md" />
```

Comment:
@anusapan you can remove this line -- all markdown files get copied over using the task at the bottom of this file